modules/gapi/include/opencv2/gapi/cpu/video.hpp

```diff
@@ -17,9 +21,209 @@ namespace cpu {
 GAPI_EXPORTS GKernelPackage kernels();
 
 } // namespace cpu
+
+struct KalmanParams
+{
+    KalmanParams()
+    {
+        statePre = Mat::zeros(dpDims, 1, type);
+        statePost = Mat::zeros(dpDims, 1, type);
+        transitionMatrix = Mat::eye(dpDims, dpDims, type);
```

Comment:
A lot of code in public headers.

Public headers should define API for Users only.
Implementation must go into "src".

/cc @dmatveev

Reply:
Done